### PR TITLE
Fixed translating and scaling issues on billboard and SDF actors

### PR DESCRIPTION
**Pull request by guaje**
This PR fixes:

 - Scaling issue on the billboard actor.
 - Translating issue on the billboard and SDF actors after upgrading to VTK 9.0.1.

|Updated Billboard actor |Updated SDF actor |
|---|---|
|Issue1|Issue2|

### CODE REVIEW

**Comment by skoudoro** [comment]
That's awesome! Thank you for fixing that. It is working like a charm on my macOS. What a tricky issue!

I will merge this PR later in the day, waiting for Travis CI.

Thank you @guaje

**Review by skoudoro — APPROVED** [review]

**Comment by dmreagan** [comment]
Nice catch, @guaje! However, it does not fix #274, as @skoudoro suggests above.

File: fury/shaders/billboard_dec.vert
Comment (dmreagan) [inline fury/shaders/billboard_dec.vert:4]: I'd like to see these follow the VTK naming convention. So this would be `centeredVertexMCVSOutput`.

File: fury/shaders/billboard_dec.vert
Comment (dmreagan) [inline fury/shaders/billboard_dec.vert:5]: and this would be `scalingFactorVSOutput`

File: fury/shaders/billboard_impl.vert
Comment (dmreagan) [inline fury/shaders/billboard_impl.vert:10]: indent

**Comment by skoudoro** [comment]
It fixes the issue @dmreagan.

A scale of 0.001 is too small to see it but if you try 0.01 for example, it works. Try to decrease the scale gradually and you will see that the effect of this fix

File: fury/shaders/billboard_impl.vert
Comment (dmreagan) [inline fury/shaders/billboard_impl.vert:11]: indent

**Comment by guaje** [comment]
> Nice catch, @guaje! However, it does not fix #274, as @skoudoro suggests above.

Yes, that's because @lenixlobo is applying the scaling on the GLSL code. That's why I didn't link this PR to that issue. I just fixed the inverted scaling on the billboard actor, but the translation issue should be fixed for both actors.

File: fury/tests/test_actors.py
Comment (dmreagan) [inline fury/tests/test_actors.py:1294]: This test doesn't appear to test anything

**Comment by skoudoro** [comment]
You see how tiny it is with 0.01
```
    scene = window.Scene()
    scene.background((1, 1, 1))
    centers = np.array([[1, 0, 0], [0, 1, 0], [0, 1, 0]])
    colors = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
    directions = np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1]])
    scales = np.array([1, 2, 3]) * 0.01
    primitive = ['sphere', 'ellipsoid', 'torus']

    sdf_actor = actor.sdf(centers, directions,
                          colors, primitive, scales)
    scene.add(sdf_actor)
    scene.add(actor.axes())
    if interactive:
        window.show(scene)
```
Capture d’écran 2020-09-15 à 18 32 44

File: fury/shaders/billboard_dec.vert
Comment (guaje) [reply fury/shaders/billboard_dec.vert:5]: I have a question regarding this variable. We are passing this variable to the fragment shader however it is not used in any of the current examples we have. Should we restrict its usage only to the vertex shader code?

File: fury/tests/test_actors.py
Comment (guaje) [reply fury/tests/test_actors.py:1294]: Thanks for noticing this

File: fury/shaders/billboard_dec.vert
Comment (dmreagan) [reply fury/shaders/billboard_dec.vert]: If it isn't used anywhere, then it is probably safe to stop passing it through. Perhaps it is just left over from some experiment and isn't really needed.

File: fury/shaders/billboard_dec.frag
Comment (skoudoro) [inline fury/shaders/billboard_dec.frag:2]: Not sure that `norm` is a good name, We can easily confuse it with normal. I personally preferred `centered`

**Review by skoudoro — COMMENTED** [review]
Any other comments? @dmreagan @lenixlobo?

Should we go ahead and merge this PR?

File: fury/shaders/billboard_dec.frag
Comment (guaje) [reply fury/shaders/billboard_dec.frag:2]: Will `normalized` be a better choice? I think `centered` is not an accurate description of what that variable is supposed to be.

Comment (skoudoro) [reply fury/shaders/billboard_dec.frag:2]: ok, it seems we have no choice to put the whole word (`normalized`)

**Comment by skoudoro** [comment]
Any other comments? I will wait until tomorrow and then merge this PR.

**Comment by skoudoro** [comment]
Ok, I'm going ahead and merge. Thank you @guaje

**Pull request merged**
